src/app/InteractionModelUtilities.h

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
could we avoid having names like utilites/helpers/support?

These seem to be 'structures' in general, but unsure if InteractionModelStructures gives me sufficient info of what this contains. Is there a more descriptive name? what do all these structures here have in common?

Or alternativel, if they do not have something specific in common, consider splitting into separate headesrs like `CommandPathParams.h`, `AttributePathParams.h` etc.

Or `InteractionModelPathParams.g` and `ClusterInfo.h`

Reply:
Done, thanks